connections/api/rpc_views.py

```diff
@@ -1,19 +1,20 @@
 import logging
-from django_oikotie.oikotie import create_apartments, create_housing_companies
-from elasticsearch_dsl import Search
 from rest_framework import status
 from rest_framework.decorators import action
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.viewsets import ViewSet
 
-from connections.etuovi.services import create_xml
+from connections.etuovi.services import create_xml as etuovi_create_xml
 from connections.etuovi.services import (
     fetch_apartments_for_sale as etuovi_fetch_apartments,
 )
-from connections.oikotie.oikotie_mapper import (
-    map_oikotie_apartment,
-    map_oikotie_housing_company,
+from connections.oikotie.services import (
+    create_xml_apartment_file,
```

Comment:
Should these be similarly separated with prefixes like `etuovi_create_xml`?

Reply:
probably should, will update